### ISSUE TITLE
Consider bundling Dazio's Control-Lock to help Linux users.

### ISSUE THREAD

**Issue by Dunbaratu** [KSP-KOS/KOS#542]
It may be a smart idea to bundle this with kOS, in the same way we can bundle ModuleManager with kOS:

http://forum.kerbalspaceprogram.com/threads/108561-0-90-%28Jan25-15%29-Control-Lock-Input-text-into-text-fields-without-issuing-commands-to-your-vessel?highlight=kos

(Check: ask Dazio if this DLL works like ModuleManager's, in the sense does it deal with multiple copies of the same DLL on the user's machine by just loading the one with the highest version and ignoring the others - an important question if we're going to consider bundling.)

**Comment by SirDiazo** [comment]
In theory it should be fine with multiple copies, but I have not actually tested it.
I made everything static which in my understanding of c# means that even if multiple copies load, they are all referencing the same static object.

Having said that, because of the icon, I did intend for a distribution mechanic similar to Blizzy's toolbar in that there should only be the 1 copy in the 001ControlLock folder.

I need to clarify the license on that and that I specifically allow redistribution like that as the intended method of bundling.

**Comment by erendrake** [comment]
I agree. It looks good and with many mods likely needing this feature we
should make it work well and share ☺
